battan40/fit-for-life
# Feature/establish sleep class test

### Is this a fix or a feature?

* Feature

### What is the change/fix?

*Sleep Test and cooresponding class files

### Where should the reviewer start?

* CLI 
* Run npm test to verify green pass
* try to break
* check with console.logs for true green passes
